services/views.py

```diff
@@ -358,9 +359,16 @@ def activate_mumble(request):
     logger.debug("activate_mumble called by user %s" % request.user)
     authinfo = AuthServicesInfoManager.get_auth_service_info(request.user)
     character = EveManager.get_character_by_id(authinfo.main_char_id)
+    # Blue members should have alliance ticker (if in alliance)
+    ticker = character.corporation_ticker
+    if EveAllianceInfo.objects.filter(alliance_id=character.alliance_id).exists():
+        alliance = EveAllianceInfo.objects.filter(alliance_id=character.alliance_id)[0]
+        alliance_ticker = alliance.alliance_ticker
+        ticker = alliance_ticker
+
     if check_if_user_has_permission(request.user, "blue_member"):
```

Comment:
Q: if you're appending the blue alliance ticker, do we really need to also have the `[BLUE]` tag on mumble? Could probably get rid of this logic here and pass both blues and members to the generic `add_user` mumble manager method.